### ISSUE TITLE
[Feature Request] Add the ability to boop local assets.

### ISSUE THREAD

**Issue by Nes370** [SteamGridDB/SGDBoop#60]
## Purpose 
It is inconvenient to apply many types of assets (horizontal grids, logos, icons, etc) to Steam through its native interface. SGDBoop offers a much easier avenue to apply these kinds of assets. However, SGDBoop currently only supports booping assets from SGDB. I believe some users may find it helpful to be able to boop their local assets, without needing to upload to SGDB first. This would help in cases where a user may not feel comfortable sharing an asset, or if an asset is not suitable to be submitted to SGDB.

## Behavior Concept
- To best mirror the convenient behavior on SGDB, a "BOOP" button is added to the right-click context menu of image files in Windows.  `Right-click image file > BOOP` Or alternatively, a file browser interface could also be used to select an image file.
- The program tries to determine the asset type the image should be applied as, derived from the file type and dimensions. Else, it prompts the user to specify which asset type it should be applied as.
- The user is then prompted to select which game to apply the asset to. The program pulls a list of games from the user's Steam library, and provides a field for the user to filter to their intended game.
- The program then applies the asset to the user's library.

**Comment by Brad7173** [comment]
I agree this would be a nice feature! https://github.com/Brad7173/kde_as_steam_os3/tree/main
I wanted to retain a full blown os but have a dedicated steam os at the same time.
Opposed to uploading grid icons to incorrect game files another idea would be to add a category for all operating systems and allow people to add custom icons. This would solve some overhead maintenance from people uploading incorrect grid icons. I've discoverd the best approach is using no widgets, desktop panels or wallpapers and loading straight into steam at startup in big picuture mode....and opposed to 500 app icons ....using latte-dock with a kde logo and opening any app from it.
